gradle/versions.gradle

```diff
@@ -15,9 +15,9 @@ version = '0.11.0-SNAPSHOT'
 ext.versions = [
 	'xtend_lib': '2.19.0',
 	'guava': '[14.0,28)',
-	'guava_orbit': '27.1.0.v20190517-1946',
+	'guava_orbit': '27.1.0.v20190517-1946',
```

Comment:
ws change intentional?

Reply:
no and i'm unable to spot what is the change. a whitespace? end of line?

Reply:
updated, i think it was a different end of line